src/dlstest/DlsClient.d

```diff
@@ -224,16 +213,13 @@ class DlsClient
 
         public void connect ( )
         {
-            this.connect_task = Task.getThis();
```

Comment:
Actually, it'd probably be better to replace this with `this.outer.raw_client.blocking.waitAllNodesConnected()`. (https://github.com/sociomantic-tsunami/swarm/blob/v4.x.x/src/swarm/neo/client/mixins/TaskBlockingCore.d#L38)

(To be honest, I do question the value of this neo client wrapper altogether as the client already has a task-blocking API, but that's an issue for another PR ;)

Reply:
Right, it might be worth to create an issue so this comment is not forgotten.

Reply:
:+1: https://github.com/sociomantic-tsunami/dlsproto/issues/38

Reply:
Yes, this task support is from the times when there was no Task support in the Neo client. I don't think there's any reason for it anymore.